### DISCUSSION TITLE
Datastore import status - non-CSV files always "waiting"

### DISCUSSION THREAD

**Discussion by stefan-korn** [GetDKAN/dkan#4102] [Q&A]
Checking the datastore import status at admin/dkan/datastore/status I see for any non csv file "waiting" for fetch, store and post import.

Reading the docs at https://dkan.readthedocs.io/en/latest/components/dkan_datastore.html I suppose that only CSV files are parsed and stored in the datastore. 

Why are non-CSV files shown in the datastore import status at all and for what are they "waiting"?

**Comment by paul-m** [comment]
How to reproduce this:

Set up a dev site with sample content:
```
ddev dkan-site-install
ddev drush pm-enable sample_content
ddev drush dkan:sample-content:create
ddev drush cron
ddev drush cron
```

Log in and visit `admin/dkan/datastore/status`

<img width="1037" alt="Screen Shot 2024-01-24 at 9 14 20 AM" src="https://github.com/GetDKAN/dkan/assets/360238/e3b945ec-1ec3-4942-84bd-442a1156d041">

This is because the dataset has resources which are not CSV files. It's true that DKAN won't localize or import files that are not CSV files.

**Comment by stefan-korn** [comment]
But does it make sense then to output these files in the datastore status? "Waiting" suggests that something is happening with these resources, but actually it is not.

**Reply by paul-m** [reply]
I was just adding repro steps...

**Comment by janette** [comment]
@stefan-korn No it does not make sense to have non-csv resources listed in this dashboard, filtering them out is in the roadmap.

**Reply by janette** [reply]
https://github.com/GetDKAN/dkan/pull/4317
